.pre-commit-config.yaml

```diff
@@ -29,3 +29,9 @@ repos:
     rev: 5.8.0
     hooks:
       - id: isort
+
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.21.2
+    hooks:
+    - id: pyupgrade
+      args: ["--py36-plus"]
```

Comment:
Since we are moving to a `.toml` for configuration, can this args be in the `pyproject.toml`? Didn't know this package existed, cool! 😃

Reply:
Hmm, honestly I don't see any mentions of config files in the pyupgrade repo other than pre-commit hook itself. So I don't think there's `.toml` support. But it's not even needed in my opinion, this tool is designed to be used in conjunction with pre-commit (they have the same author), maintainer even explicitly stated that running pyupgrade directly on multiple files/directory will not be supported.
P.S. To run it through pre-commit on all files you'd use `pre-commit run pyupgrade -a`.

Reply:
Ah got it. Sounds good to me.